server/routes/aboutPerson/AboutPersonController.ts

```diff
@@ -32,6 +32,24 @@ export default class AboutPersonController {
       const pageId = 'about'
       const readWrite = req.services.sessionService.getAccessMode() === AccessMode.READ_WRITE
 
+      const areas = [...assessmentAreas.lowScoring, ...assessmentAreas.highScoring, ...assessmentAreas.other]
+
+      for (const area of areas) {
```

Comment:
I think instead of duplicating the bulk of the return statement you could do something like `let returnPage = 'pages/about'; if(areas.some(area => area.isMissingInformation)) { returnPage = 'pages/about-not-complete' }` and then just do res.render(returnPage, ...

Reply:
Done!